### PR TITLE
pypy3: Add pypy3.6 v7.1.1

### PR DESCRIPTION
**Pull request by TheCjw**
Just add pypy3.6.

### CODE REVIEW

File: bucket/pypy3.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "7.1.1",
+    "license": "MIT",
+    "extract_dir": "pypy3.6-v7.1.1-win32",
+    "url": "https://bitbucket.org/pypy/pypy/downloads/pypy3.6-v7.1.1-win32.zip",
+    "homepage": "https://pypy.org",
+    "hash": "8b513b254de5f31890f5956569de9aec3a0a91d7aba72fc89d66901f4a8ccf49",
+    "bin": [
```
Comment (Ash258) [inline bucket/pypy3.json]:
```suggestion
    "bin": [
        "pypy3.exe",
        "pypy3w.exe"
    ],
```

File: bucket/pypy3.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "7.1.1",
+    "license": "MIT",
+    "extract_dir": "pypy3.6-v7.1.1-win32",
```
Comment (Ash258) [inline bucket/pypy3.json]: No description

File: bucket/pypy3.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        ]
+    ],
+    "checkver": {
+        "url": "https://bitbucket.org/pypy/pypy/downloads/",
+        "re": "pypy3.6-v([\\d.]+)-win32"
+    },
+    "autoupdate": {
+        "url": "https://bitbucket.org/pypy/pypy/downloads/pypy3.6-v$version-win32.zip",
+        "extract_dir": "pypy3.6-v$version-win32"
+    }
```
Comment (Ash258) [inline bucket/pypy3.json]: Add hash extraction from https://pypy.org/download.html

Should be enough
```suggestion
        "hash": {
            "url": "https://pypy.org/download.html"
        }
    }
```

**Comment by TheCjw** [comment]
Fixed and tested.

**Comment by Ash258** [comment]
Nice 👍
Do same for that 2.7 version PR please.

**Comment by TheCjw** [comment]
All fix now.

**Comment by Ash258** [comment]
You could try to run checkver with `-f` and see if everything is OK.
It would check for new version, automatically change versions and format. Push changes if there are any or do nothing if it is all OK

**Comment by TheCjw** [comment]
Thanks, I didn't notice `-f` switch before.

**Pull request merged**